app/lib/widgets/TypeIndicator.dart

```diff
@@ -56,189 +56,192 @@ class TypeIndicator extends StatefulWidget {
 class _TypeIndicatorState extends State<TypeIndicator>
     with TickerProviderStateMixin {
   late double stackingWidth;
-  late AnimationController _appearanceController;
-  late AnimationController _animatedCirclesController;
-  late Animation<double> _indicatorSpaceAnimation;
-  late Animation<Offset> _firstCircleOffsetAnimation;
-  late Animation<Offset> _secondCircleOffsetAnimation;
-  late Animation<Offset> _thirdCircleOffsetAnimation;
+  late AnimationController appearanceController;
```

Comment:
Imo this should be declared private. We would want restricted access here.

Reply:
Please read this comment.
https://github.com/effektio/effektio/pull/276#issuecomment-1279790599